recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
-  sha256: 86201502fab9c3bba8aa6ffaac0f465b0bcdb75ec75fd157eda8b96352fb3cf3
+  sha256: 213b573c71d4424d33a660d3aa2419a7ef74daf8902cdc2daffe8900844f4b2e
```

Comment:
I confirmed that this matches [here](https://pypi.org/project/optuna-integration/4.2.0/#optuna_integration-4.2.0.tar.gz).